keras/metrics.py

```diff
@@ -91,6 +91,32 @@ def matthews_correlation(y_true, y_pred):
 
     return numerator / (denominator + K.epsilon())
 
+
+def fmeasure(y_true, y_pred, beta=1):
```

Comment:
No, because `beta` is a parameter and not fixed to one. [fbeta_score](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.fbeta_score.html) has precedent. Would that work?